python/gherkin3/token_matcher.py

```diff
@@ -17,6 +20,41 @@ def reset(self):
         self._indent_to_remove = 0
         self._active_doc_string_separator = None
 
+    def _change_dialect(self, dialect_name, location=None):
```

Comment:
I agree with @brasmusson - I don't see how the new proposed ordering improves readability or consistency with the other implementations.